### PR TITLE
Re-organised documentation of the sprint board

### PR DESCRIPTION
**Pull request by oscarhirst**
Re-organised content about the sprint board to reduce repetition and appropriately group the information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,23 +138,28 @@ During this meeting the team engages in the following activities to prepare for
...
 
 Tasks remain in the verify column until the code has been reviewed by another team member. Once tasks have been reviewed, they can be moved to the ‘done’ column. Sometimes it is possible to review a single task, but in practice, an entire story may be reviewed as a whole. If changes are requested as part of the review, these should be recorded onto new Post-its and placed in the ‘to do’ column.
 
-To minimise the quantity of partially complete stories at the end of a sprint, a limit is imposed on the number of work in progress stories. Currently, this limit is set at 3 stories.
+Occasionally, stories may be deemed too small to be broken down into fine-grained tasks. These get grouped into a ‘Single Card Stories’ lane at the top of the board.
+
+To minimise the quantity of partially complete stories at the end of a sprint, a limit is imposed on the number of 'work in progress stories' (the number of stories currently being worked on). Currently, this limit is set at 3 stories.
```
Comment (lparkermg) [inline README.md]: Should we include the fact that having this limit encourages pair programming among team members etc?

File: README.md [hunk 2]
```diff
@@ -138,23 +138,28 @@ During this meeting the team engages in the following activities to prepare for
...
 
-During the sprint, the team’s progress on the tasks generated during the sprint kick-off is tracked using Post-it notes on a [large physical whiteboard](#sprint-board). This gives the entire team visibility of sprint progress at all times.
+The development team has a large, physical whiteboard in their workspace called the _sprint board_.
+
+During a sprint, the team’s progress on the tasks generated during the sprint kick-off is tracked using Post-it notes on the sprint board. This gives the entire team visibility of sprint progress at all times.
 
 ![The Cognisant development team sprint board (captured mid-sprint)](/images/sprint-board.jpeg "The sprint board (captured mid-sprint)")
 
-The board is arranged into columns named ‘to do’, ‘doing’, ‘verify’, and ‘done’. Tasks belonging to each story in the sprint are grouped into horizontal lanes signified by a title card on the far left.
+Kanban columns (columns named ‘to do’, ‘doing’, ‘verify’, and ‘done’) occupy most of the sprint board. The team uses these to track the progress of work during a sprint.
```
Comment (jbradford) [inline README.md]: Kanban and Scrum are different things, I'm not sure it makes sense to complicate our documentation with the term when it doesn't really mean anything in this context, it just happens that a Kanban board could have similar column names.

File: README.md [hunk 3]
```diff
@@ -251,22 +263,12 @@ Temporary channels for ad-hoc discussions are prefixed with `temp`, for example,
...
-Kanban columns occupy most of the sprint board. The team uses these to track the progress of work during a sprint. See [Tracking Progress](#Tracking-Progress) for more details.
-
-Each task is represented by a Post-it on the board. These Post-its move through the columns on the board throughout the sprint. By the end of a sprint, most (if not all) of the task Post-its will have migrated from the ‘To do’ column through the ‘Doing’ and ‘Verify’ columns to the ‘Done’ column.
-
-#### Distractions
-
-There is a reserved area on the sprint board for distractions. Development team members add a Post-it to this area whenever they deviate from sprint-related work. Team members add the duration of the distraction and their initials to the Post-it. The development team reviews distractions during the [retrospective](#Retrospective).
-
-#### Retrospective Goals
+It has two main sections:
```
Comment (jbradford) [inline README.md]: We should probably keep the definitions here, then link to them in the tracking progress section. We now have 2 sections about the sprint board.

File: README.md [hunk 4]
```diff
@@ -214,6 +219,13 @@ Any possible process improvements agreed upon by the team get trialled in a futu
 
 An example of a process improvement derived from a retrospective meeting is [the work in progress limit](https://github.com/cognisant/Docs.DevelopmentProcess/blob/dev-with-retrospective/docs/work-in-progress-limit.md).
 
+##### Retrospective Goals
```
Comment (jbradford) [inline README.md]: Seems like a section on retrospective goals might have made more sense in a separate PR.

File: README.md (hunk 1 above)
Comment (oscarhirst) [reply README.md]: I have added a link here to more information about the WiP limit

File: README.md (hunk 2 above)
Comment (oscarhirst) [reply README.md]: I copied the reference to Kanban from docs Rich wrote previously.

Comment (oscarhirst) [reply README.md]: Have now removed it

File: README.md [hunk 5]
```diff
@@ -251,22 +256,12 @@ Temporary channels for ad-hoc discussions are prefixed with `temp`, for example,
 Some channels are used with third-party integrations to provide notifications. These are prefixed with `bot`, for example, `#bot-monitoring`.
 
 ### Sprint Board
+The development team has a large, physical whiteboard in their workspace called the sprint board, it is used to track different aspect of the sprint.
```
Comment (iBeizsley) [inline README.md]: This reads slightly oddly. One of the following would be better:

`called the sprint board. It is used`

or

`called the sprint board, which is used`

File: README.md [hunk 6]
```diff
@@ -251,22 +256,12 @@ Temporary channels for ad-hoc discussions are prefixed with `temp`, for example,
...
-
-#### Distractions
-
-There is a reserved area on the sprint board for distractions. Development team members add a Post-it to this area whenever they deviate from sprint-related work. Team members add the duration of the distraction and their initials to the Post-it. The development team reviews distractions during the [retrospective](#Retrospective).
-
-#### Retrospective Goals
+It has two main sections:
 
-There is an area on the sprint board for highlighting the current retrospective goals. These act as constant reminders to the team about what the agreed goals are for improving the process.
+* Sprint Progress Columns (which occupy most of the sprint board)
```
Comment (iBeizsley) [inline README.md]: Is it worth noting that this consists of retrospective, tech debt, and current sprint stories?

File: README.md (hunk 5 above)
Comment (rcknight) [reply README.md]: This sentence is duplicated from the changed section above. I think the tools section should be limited to a description of the kind of board, the column markers etc that we use (with links to websites).

**Pull request merged**